src/vue/Components/Accounts/AccountList.vue

```diff
@@ -1,8 +1,8 @@
 <template>
     <div class="account-list">
         <translate tag="h3" say="AccountList">
-            <icon icon="user-plus" font="solid" @click="showCreateForm" v-if="!addAccount"/>
-            <icon icon="qrcode" font="solid" @click="openPassLinkScan"/>
+            <icon icon="user-plus" font="solid" @click="showCreateForm" v-if="!addAccount" title="Add user account" />
+            <icon icon="qrcode" font="solid" @click="openPassLinkScan" title="Add user account via QR-code" />
```

Comment:
The title is a translatable attribute. Please use a language tag and add the translation in `src/platform/generic/_locales/en/messages.json` so it can be translated for other languages